Makefile

```
@@ -669,6 +669,14 @@ test-capd-junit: $(SETUP_ENVTEST) $(GOTESTSUM) ## Run unit and integration tests
 kind-cluster: ## Create a new kind cluster designed for development with Tilt
 	hack/kind-install-for-capd.sh
 
+.PHONY: kind-reset
+kind-reset: ## Destroys the "$CAPI_KIND_CLUSTER_NAME" kind cluster.
+	kind delete cluster --name="${CAPI_KIND_CLUSTER_NAME:-"capi-test"}|| true
```

```suggestion
	kind delete cluster --name="${CAPI_KIND_CLUSTER_NAME:-"capi-test"}"|| true
```

Also, it doesnt look like default works properly here:
```
$ CAPI_KIND_CLUSTER_NAME="test-cluster" make kind-reset
kind delete cluster --name=""|| true
Deleting cluster "" ...
```

---

I don't think the bash syntax works in Makefiles the same way (cf. somewhere above `PROD_REGISTRY ?= k8s.gcr.io/cluster-api`)